source/utils/blockUntilConditionMet.py

```diff
@@ -0,0 +1,69 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2020-2022 NV Access Limited
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+
+from time import (
+	perf_counter as timer,
+	sleep,
+)
+from typing import (
+	Any,
+	Callable,
+	Optional,
+	Tuple,
+)
+
+
+EvaluatorWasMetT = bool
+GetValueResultT = Any
+
+DEFAULT_INTERVAL_BETWEEN_EVAL_SECONDS = 0.3

```

Comment:
I think these values should have been left as is, as they didn't need to match the system tests.
This doesn't matter, as both sets of values are going to be changed in #14324